app/main.py

```diff
@@ -86,128 +86,114 @@
 }
 
 
-def battle(knightsConfig):
+class Knight:
+    def __init__(self,
+                 name: str,
+                 power: int,
+                 hp: int,
+                 armour: dict,
+                 weapon: dict,
+                 potion: dict) -> None:
+        self.name = name
+        self.hp = hp
+        self.power = power
+        self.armour = armour
+        self.weapon = weapon
+        self.potion = potion
+        self.protection = 0
+
+    def preparation(self) -> None:
+        # apply armour
+        for i in self.armour:
```

Comment:
`i` is a bad name. rename please

Reply:
pls resolve comment

Reply:
`i is a bad name. rename please`
rename i to part

Reply:
@ladno338 not fixed